projects/ccf-rui/src/app/shared/components/organ-selector/organ-selector.component.ts

```diff
@@ -42,19 +47,19 @@ export class OrganSelectorComponent {
    * List of organs in the carousel
    */
   organList: OrganInfo[] = [
-    {src: 'app:bladder', name: 'Bladder'},
-    {src: 'app:brain', name: 'Brain'},
-    {src: 'app:colon', name: 'Colon'},
-    {src: 'app:heart', name: 'Heart'},
-    {src: 'app:kidney', name: 'Kidney'},
-    {src: 'app:liver', name: 'Liver'},
-    {src: 'app:lung', name: 'Lung'},
-    {src: 'app:lymph_nodes', name: 'Lymph Nodes'},
-    {src: 'app:ovaries', name: 'Ovaries'},
-    {src: 'app:small_intestine', name: 'Small Intestine'},
-    {src: 'app:spleen', name: 'Spleen'},
-    {src: 'app:stomach', name: 'Stomach'},
-    {src: 'app:thymus', name: 'Thymus'}
+    {src: 'app:bladder', name: 'Bladder', hasData: false},
```

Comment:
Change this so colon, heart, kidney, and spleen are the items with with data and put them first in the list so they don't have to scroll through the other items.

Reply:
Also change the property to disabled and only have the ones disabled have a property: `disabled: true`.